airflow/hooks/postgres_hook.py

```diff
@@ -111,3 +126,34 @@ def _serialize_cell(cell, conn):
         :rtype: object
         """
         return cell
+
+    def get_iam_token(self, conn):
+        """
+        Uses AWSHook to retrieve a temporary password to connect to Postgres
+        or Redshift. Port is required. If none is provided, default is used for
+        each service
+        """
+        from airflow.contrib.hooks.aws_hook import AwsHook
+
+        redshift = conn.extra_dejson.get('redshift', False)
+        aws_conn_id = conn.extra_dejson.get('aws_conn_id', 'aws_default')
+        aws_hook = AwsHook(aws_conn_id)
+        login = conn.login
+        if conn.port is None:
+            port = 5439 if redshift else 5432
+        else:
+            port = conn.port
+        if redshift:
+            cluster_identifier = conn.extra_dejson.get('cluster-identifier', conn.host.split('.')[0])
```

Comment:
This could do with a comment showing an example conn.host and what value we want out of it.

Reply:
agree